### ISSUE TITLE
Obj-C error when trying to Init

### ISSUE THREAD

**Issue by scottishwildcat** [mathieudutour/git-sketch-plugin#221]
### How did it happen?
1. Installed plugin
2. Advanced > Init git repo
3. Alert: "Error: An Obj-C exception occurred."

### Error log

```
{
	"nativeException": {},
	"line": 195,
	"column": 37,
	"sourceURL": "/Users/calum/Library/Application Support/com.bohemiancoding.sketch3/Plugins/Git.sketchplugin/Contents/Sketch/Init.js"
}
```

**Comment by konogen1** [comment]
I also encountered this problem.
I tried another project file, but I also encountered this.

**Comment by mathieudutour** [comment]
fixed in 0.12
